src/components/Footer.js

```diff
@@ -3,62 +3,59 @@ import '../styles/Footer.css';
-              <li className="p-2">
-                <a className="text-white" href="#footer2">
-                  Footer 2
-                </a>
-              </li>
-              <li className="p-2">
-                <a className="text-white" href="#footer3">
-                  Footer 3
-                </a>
-              </li>
-            </ul>
-            <ul className="col col-lg-4 list-unstyled">
-              <li className="p-2">
-                <a className="text-white" href="#footer4">
-                  Footer 4
-                </a>
-              </li>
-              <li className="p-2">
-                <a className="text-white" href="#footer5">
-                  Footer 5
-                </a>
-              </li>
-              <li className="p-2">
-                <a className="text-white" href="#footer6">
-                  Footer 6
-                </a>
-              </li>
-            </ul>
-          </div>
+    <footer className="container-fluid p-3 mt-5">
+      <div className="row">
+        <div className="col col-lg-6 text-center text-lg-left">
+          <ul className="p-3 list-unstyled">
+            <li className="p-2">
+              <a
+                className="text-white"
+                href="mailto:learnwitheden@gmail.com"
+                rel="noopener noreferrer"
+                target="blank"
+              >
+                Contact Us: learnwitheden@gmail.com
+              </a>
+            </li>
+            <li className="p-2">
+              <a className="text-white" href="#footer2">
+                Terms & Privacy
+              </a>
+            </li>
+          </ul>
+        </div>
 
-          <div className="col-lg-6 text-center">
-            <a href="#twitter">
-              <i className="fa fa-twitter rounded-circle fa-2x bg-white m-4 p-3" />
-            </a>
-            <a href="#facebook">
-              <i className="fa fa-facebook-square rounded-circle fa-2x bg-white m-4 p-3" />
-            </a>
-            <a href="#instagram">
-              <i className="fa fa-instagram rounded-circle fa-2x bg-white m-4 p-3" />
-            </a>
-            <a href="#linkedin">
-              <i className="fa fa-linkedin rounded-circle fa-2x bg-white m-4 p-3" />
-            </a>
-          </div>
+        <div className="col-lg-6 text-center">
+          <a
+            href="https://twitter.com/PlaywithEden"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <i className="fa fa-twitter rounded-circle fa-2x bg-white m-4 p-3" />
+          </a>
+          <a
+            href="https://m.facebook.com/PlayWithEden/?modal=admin_todo_tour"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <i className="fa fa-facebook-square rounded-circle fa-2x bg-white m-4 p-3" />
+          </a>
+          <a
+            href="https://www.instagram.com/invites/contact/?i=1q6wz6xs4q4dc&utm_content=g2oxjvo"
```

Comment:
Can you double check the link, this one is not working for me. This seems like an invitation link.